hugo/content/capabilities/documentation-quality/index.md

```diff
@@ -30,6 +30,9 @@ What may be more exciting, however, is how documentation quality interacts with
 
 This interaction and impact applies to every single technical capability we studied in 2022. In the table below, check out the lift to organizational performance as various technical capabilities are implemented. The lift is significantly amplified for teams with above-average documentation quality.
 
+
+#### Lift to organizational performance as technical capabilities are implemented
```

Comment:
Should this be ###

Reply:
discussed offline. we looked at `###`, `####`, and **bold** as options. Decided that **bold** was the best option with a trailing `:` to indicate that this line is an intro to the table.